doc/UsersGuide/source/tables/fv3diagtable.csv

```diff
@@ -0,0 +1,416 @@
+﻿Variable,Description
+cldfra2d,Instantaneous 2D (max in column) fraction
+total_albedo,Total sky albedo at top of atmosphere
+lwp_ex,Total liquid water path from explicit microphysics
+iwp_ex,Total ice water path from explicit microphysics
+lwp_fc,Total liquid water path from cloud fraction scheme
+iwp_fc,Total ice water path from cloud fraction scheme
+ALBDO_ave,Surface albedo
+DLWRF,Surface downward longwave flux
+DLWRFI,Instantanous surface downward longwave flux
+ULWRF,Surface upward longwave flux
+DSWRFItoa,Instantaneous top of atmosphere downward shortwave flux
+USWRFtoa,Instantaneous top of atmosphere upward shortwave flux
+ULWRFtoa,Instantaneous top of atmosphere upward longwave flux
+ULWRFI,Instantaneous surface upward longwave flux
+DSWRF,Averaged surface downward shortwave flux
+DSWRFI,Instantaneous surface downward shortwave flux
+USWRF,Averaged surface upward shortwave flux
+USWRFI,Instantaneous surface upward shortwave flux
+duvb_ave,UV-B downward solar flux
+cduvb_ave,Clear sky UV-B downward solar flux
+vbdsf_ave,Visible beam downward solar flux
+vddsf_ave,Visible diffuse downward solar flux
+nbdsf_ave,Near IR beam downward solar flux
+nddsf_ave,Near IR diffuse downward solar flux
+csulf_avetoa,Clear sky upward longwave flux at top of atmosphere
+csusf_avetoa,Clear sky upward shortwave flux at top of atmosphere
+csdlf_ave,Clear sky downward longwave flux
+csusf_ave,Clear sky upward shortwave flux
+csdsf_ave,Clear sky downward shortwave flux
+csulf_ave,Clear sky upward longwave flux
+DSWRFtoa,Top of atmosphere downward shortwave flux
+USWRFtoa,Top of atmosphere upward shortwave flux
+ULWRFtoa,Top of atmosphere upward longwave flux
+TCDC-aveclm,Atmosphere column total cloud cover
+TCDC_avebndcl,Boundary layer cloud layer total cloud cover
+TCDCcnvcl,Convective cloud layer total cloud cover
+PREScnvclt,Pressure at convective cloud top level
+PREScnvclb,Pressure at convective cloud bottom level
+TCDC_avehcl,High cloud level total cloud cover
+PRES_avehct,Pressure high cloud top level
+PRES_avehcb,Pressure high cloud bottom level
+TEMP_avehct,Temperature at high cloud top level
+TCDC_avemcl,Mid-cloud level total cloud cover
+PRES_avemct,Pressure at middle cloud top level
+PRES_avemcb,Pressure at middle cloud bottom level
+TEMP_avemct,Temperature middle cloud top level
+TCDC_avelcl,Low cloud level total cloud cover
+PRES_avelct,Pressure low cloud top level
+PRES_avelcb,Pressure at low cloud bottom level
+TEMP_avelct,Temperature at low cloud top level
+AOD_550,Total aerosol optical depth at 550 nm
+DU_AOD_550,Dust aerosol optical depth at 550 nm
+BC_AOD_550,Soot aerosol optical depth at 550 nm
+OC_AOD_550,Water soluble aerosol optical depth at 550 nm
+SU_AOD_550,Suspended organic aerosol optical depth at 550 nm
+SS_AOD_550,Salt aerosol optical depth at 550 nm
+aod,Total aerosol optical depth at 550 nm

```

Comment:
Lines 950-963 include a variable called `fluxr_` --> 'fluxr diagnostic - GFS radiation' that doesn't seem to be included here.